.env.example

```
@@ -131,6 +146,20 @@ KONG_HTTPS_PORT=8443
+############
+# [required]
+# Storage - Configuration for the Supabase Storage service.
+# Upstream Supabase moved these out of docker-compose.yml into .env (storage-api v1.48+).
+# Leaving REGION unset causes the storage container to crash with "region is missing".
+# These are safe local defaults; the S3_PROTOCOL keys are the upstream demo values.
+############
+
+REGION=stub
+GLOBAL_S3_BUCKET=stub
+STORAGE_TENANT_ID=stub
+S3_PROTOCOL_ACCESS_KEY_ID=625729a08b95bf1b7ff351a663f3a23c
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_PROTOCOL_ACCESS_KEY_ID key should go before the STORAGE_TENANT_ID key

(UnorderedKey)